types/luxon/src/datetime.d.ts

```diff
@@ -943,7 +1001,7 @@ export class DateTime {
      * @example
      * DateTime.now().toLocaleString({ hour: '2-digit', minute: '2-digit', hourCycle: 'h23' }); //=> '11:32'
      */
-    toLocaleString(formatOpts?: DateTimeFormatOptions, opts?: LocaleOptions): string;
+    toLocaleString(formatOpts?: DateTimeFormatPreset | DateTimeFormatOptions, opts?: LocaleOptions): string;
```

Comment:
Why was this type changed at all? Should the presets always be valid `DateTimeFormatOptions` as well?

Reply:
I believe the intent was to provide better typing for presets. I've read a book, the author rewrote ending 39 times, this applies to our code IMO
#55750